pandas/tests/frame/test_convert_to.py

```diff
@@ -175,7 +172,34 @@ def test_to_records_index_name(self):
     def test_to_records_with_unicode_index(self):
         # GH13172
         # unicode_literals conflict with to_records
-        result = DataFrame([{u'a': u'x', u'b': 'y'}]).set_index(u'a')\
+        result = DataFrame([{u'a': u'x', u'b': 'y'}]).set_index(u'a') \
             .to_records()
         expected = np.rec.array([('x', 'y')], dtype=[('a', 'O'), ('b', 'O')])
         tm.assert_almost_equal(result, expected)
+
+    def test_to_records_with_tz_gmt(self):
+        data = [datetime.datetime.now(pytz.timezone('UTC')) for _ in range(10)]
+        df = DataFrame({'datetime': data})
+        df.set_index('datetime', inplace=True)
+        dfgmt = df.tz_convert("GMT")
+        tm.assertIsNot(df.to_records()['datetime'][0].tzinfo,
```

Comment:
Can you also simplify the construction of the test data using pandas constructs?

Eg 

```
data = [datetime.datetime.now(pytz.timezone('UTC')) for _ in range(10)]
df = DataFrame({'datetime': data})
df.set_index('datetime', inplace=True)
```

can be done with (for similar object, not exact same values of course)

```
df = DataFrame(index=date_range('2016-01-01', periods=10, freq='S', tz='UTC'))
```